gpflow/posterior.py

```diff
@@ -189,14 +466,14 @@ def predict_f(
         M, L, N, K = tf.unstack(tf.shape(Kuf), num=Kuf.shape.ndims, axis=0)
         Kuf = tf.reshape(Kuf, (M * L, N * K))
 
-        Knn = self.kernel(Xnew, full_cov=full_cov, full_output_cov=full_output_cov)
+        Kff = self.kernel(Xnew, full_cov=full_cov, full_output_cov=full_output_cov)
```

Comment:
Should the `full_output_cov` argument be removed?

Reply:
I don't think so; it changes the shape of Knn....